app/views/macros/payment-request-header.njk

```diff
@@ -4,23 +4,29 @@
         <thead class="govuk-table__head">
           <tr class="govuk-table__row">
             <th scope="col" class="govuk-table__header govuk-body-s">Scheme</th>
+            <th scope="col" class="govuk-table__header govuk-body-s">Scheme year</th>
             <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">FRN</th>
             <th scope="col" class="govuk-table__header govuk-body-s">Agreement number</th>
             <th scope="col" class="govuk-table__header govuk-body-s">Invoice</th>
             <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Request</th>
             <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Total amount</th>
-            <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Ledger</th>
+            {% if not paymentRequestData.manualLedgerChecks%}
+              <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Ledger</th>
+            {% endif %}
           </tr>
         </thead>
         <tbody class="govuk-table__body">
           <tr class="govuk-table__row">
             <td class="govuk-table__cell govuk-body-s">{{paymentRequestData.schemes.name}}</td>
+            <td class="govuk-table__cell govuk-body-s">{{paymentRequestData.marketingYear}}</td>
```

Comment:
Same comment on marketing year

Reply:
Dont think this is required as above comment.